CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+How to contribute
+=================
+We are happy to accept contributions. To accept a contribution we require:
+
+* A developer certificate of origin: see http://developercertificate.org/
```

Comment:
Where did the template for this contributing.md file come from? Do we really use DC?

Reply:
We don't always use the DCO, but when we do it saves our bacon. The template is from somewhere, but most of our projects have this or similar: https://github.com/Pelagicore/softwarecontainer/blob/master/CONTRIBUTING.md

Reply:
This was copied from the meta-pelux's contributing.md file. But perhaps I should follow the templates in softwarecontainer as described by Jeremiah

Reply:
No problem imho, if they are the same, they are the same.